packages/talker_http_logger/lib/talker_http_logger.dart

```diff
@@ -4,20 +4,26 @@ import 'dart:convert';
 
 import 'package:http_interceptor/http_interceptor.dart';
 import 'package:talker/talker.dart';
+import 'package:talker_http_logger/talker_http_logger_settings.dart';
 
 class TalkerHttpLogger extends InterceptorContract {
-  TalkerHttpLogger({Talker? talker}) {
+  TalkerHttpLogger(
+      {Talker? talker, this.settings = const TalkerHttpLoggerSettings()}) {
     _talker = talker ?? Talker();
   }
 
   late Talker _talker;
 
+  /// [TalkerHttpLogger] settings and customization
+  TalkerHttpLoggerSettings settings;
+
```

Comment:
I understand that this feature is same for talker_http and talker_dio but can we create separated PR with ``TalkerHttpLoggerSettings`` for talker_http. And ``hiddenHeaders `` will be added for ``TalkerHttpLoggerSettings`` in this new Pull-request ?